REST_API/get-session-commands.js

```diff
@@ -0,0 +1,19 @@
+const request = require('request');
+
+const username='<KOBITON_USERNAME>';
+const apikey='<KOBITON_APIKEY>';
+
+var session = '<SESSION_ID>';
+request({
+  url: `https://api.kobiton.com/v1/sessions/${session}/commands`,
```

Comment:
What if the user want to access other pages of the command list?
Ref: https://api.kobiton.com/docs/#get-session-commands
```
Parameter: page
The page number of commands, start from 1. Each page contains 20 commands.
```